CMakeLists.txt

```diff
@@ -38,6 +38,10 @@ option(BUILD_EXAMPLES "Build s2 documentation examples." ON)
 
 feature_summary(WHAT ALL)
 
+# TODO(jmr): Allow use of installed abseil; needs Findabsl.cmake or similar.
```

Comment:
abseil already supports the find_packge mode, you can see https://github.com/abseil/abseil-cpp/releases.